src/net/NodeHttpClient.ts

```diff
@@ -13,6 +14,8 @@ const defaultHttpsAgent = new https.Agent({keepAlive: true});
  * requests.`
  */
 class NodeHttpClient extends HttpClient {
+  _agent: any;
```

Comment:
Can we mark it as `http.Agent | https.Agent`? Not sure what's our dependency on @types/node package.

Reply:
Yep, looks like `http.Agent | https.Agent` does the trick.